webapp-django/crashstats/base/tests/test_finders.py

```diff
@@ -0,0 +1,33 @@
+import copy
+
+from nose.tools import assert_raises
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management import call_command
+
+from crashstats.base.tests.testbase import DjangoTestCase
+
+
+class LeftoverPipelineFinder(DjangoTestCase):
+    """Test our custom staticfiles finder class."""
+
+    def test_missing_css_source_file(self):
+        busted_pipeline = copy.deepcopy(settings.PIPELINE)
+        # Doesn't matter which key we chose to bust
+        for key in busted_pipeline['STYLESHEETS']:
```

Comment:
Updated now. 

The point about supporting lists *and* generators is principle but not important enough here. It's just a test. In fact, it's kinda nice in this case to use `[0]` because you know for sure there is at least one. 

Updated the code now.